src/main/java/com/google/planningworks/cdap/plugins/source/AnaplanExportConfig.java

```diff
@@ -0,0 +1,506 @@
+      + " of parallelism, but will require more resources and overhead.")
+  private Long maxSplitSize;
+
+  @Macro
+  @Nullable
+  @Description("Minimum size of each partition used to read data. ")
+  private Long minSplitSize;
+
+  @Macro
+  @Nullable
+  @Description(
+    "Output field to place the path of the file that the record was read from. "
+      + "If not specified, the file path will not be included in output records. "
+      + "If specified, the field must exist in the output schema as a string.")
+  private String pathField;
+
+  @Macro
+  @Description(
+    "Format of the data to read. Supported formats are 'avro', 'blob', 'csv', 'delimited',"
+      + " 'json', 'parquet', 'text', and 'tsv'.")
+  private String format;
+
+  @Macro
+  @Nullable
+  @Description(
+    "Output schema. If a Path Field is set, it must be present in the schema as a string.")
+  private String schema;
+
+  @Macro
+  @Nullable
+  @Description(
+    "Whether to only use the filename instead of the URI of the file path when a path field is"
+      + " given. The default value is false.")
+  private Boolean filenameOnly;
+
+  @Macro
+  @Nullable
+  @Description(
+    "Regular expression that file paths must match in order to be included in the input. The full"
+      + " file path is compared, not just the file name.If no value is given, no file filtering"
+      + " will be done. See"
+      + " https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html for more"
+      + " information about the regular expression syntax.")
+  private String fileRegex;
+
+  @Macro
+  @Nullable
+  @Description(
+    "Whether to recursively read directories within the input directory. The default is false.")
+  private Boolean recursive;
+
+  @Macro
+  @Nullable
+  @Description(
+    "The delimiter to use if the format is 'delimited'. The delimiter will be ignored if the"
+      + " format is anything other than 'delimited'.")
+  private String delimiter;
+
+  @Macro
+  @Nullable
+  @Description(
+    "Whether to skip the first line of each file. Supported formats are 'text', 'csv', 'tsv', "
+      + "'delimited'. Default value is false.")
+  private Boolean skipHeader;
+
+  @Macro
+  @Nullable
+  @Description("File encoding for the source files. The default encoding is 'UTF-8'")
+  private String fileEncoding;
+
+  // This is a hidden property that only exists for wrangler's parse-as-csv that uses the header as
+  // the schema. When this is true and the format is text, the header will be the first record
+  // returned by every record reader
+  @Nullable
+  private Boolean copyHeader;
+
+  @Macro
+  @Nullable
+  @Description(
+    "Whether the data file is encrypted. If it is set to 'true', a associated metadata file needs"
+      + " to be provided for each data file. Please refer to the Documentation for the details"
+      + " of the metadata file content.")
+  private Boolean encrypted;
```

Comment:
remove unless you have requirements that the temporary data needs to be encrypted.